cluster/manifests/01-admission-control/config.yaml

```diff
@@ -16,7 +16,6 @@ data:
   pod.parent-resource-hash.enable: "{{ .Cluster.ConfigItems.teapot_admission_controller_parent_resource_hash }}"
 {{- if or (eq .Cluster.ConfigItems.teapot_admission_controller_service_account_iam "true") (eq .Cluster.ConfigItems.teapot_admission_controller_service_account_iam_userdata "true") }}
   pod.service-account-iam.enable: "true"
-  pod.service-account-iam.base-aws-account-id: "{{ accountID .Cluster.InfrastructureAccount }}"
```

Comment:
This we should keep, otherwise it's going to be harder to roll out the new version.